tpm/tpm_test.go

```diff
@@ -232,6 +233,35 @@ func TestResizeableSlice(t *testing.T) {
 	}
 }
 
+func TestNVDefineSpace(t *testing.T) {
+	rwc := openTPMOrSkip(t)
+	defer rwc.Close()
+	index := tpmutil.Handle(0x40000001)
+	permissions := []uint32{nvPerWriteSTClear}
+	length := uint32(64)
+	ownerAuth := getAuth(ownerAuthEnvVar)
+	if _, err := NVDefineSpace(rwc, index, length, permissions, ownerAuth); err != nil {
+		t.Fatal("Couldn't define space:", err)
+	}
+
+}
+
+func TestNVWriteValue(t *testing.T) {
+	rwc := openTPMOrSkip(t)
+	defer rwc.Close()
+	index := uint32(0x40000001)
+	ownerAuth := getAuth(ownerAuthEnvVar)
+	r := make([]byte, 20)
+	_, err := rand.Read(r)
+	if err != nil {
+		t.Fatal("Couldn't read random:", err)
+	}
+
+	if _, err = NVWriteValue(rwc, index, 0, r, ownerAuth); err != nil {
```

Comment:
Please also read back the value and confirm that it matches the value that was written. That will give an end-to-end test of the consistency of NVWriteValue and NVReadValue.